.circleci/config.yml

```
@@ -220,6 +220,7 @@ commands:
           condition: << parameters.linux >>
           steps:
             - run: sudo apt-get update
+            - run: sudo apt-get remove needrestart
             - run: sudo apt-get install --no-install-recommends -y valgrind ocl-icd-opencl-dev libssl-dev libhwloc-dev nvidia-cuda-toolkit g++-10
```

```suggestion
            - run: DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -y valgrind ocl-icd-opencl-dev libssl-dev libhwloc-dev nvidia-cuda-toolkit g++-10
```

this should be a safer and more straightforward way to achieve this